rj_msgs/msg/RobotState.msg

```diff
@@ -0,0 +1,9 @@
+# This message contains data that comes from the vision system
+# including position data and which camera this robot was seen by and
+# what time it was last seen.
+# TODO(?): Revisit the "which camera" part. It seems to be missing in WorldState.hpp.
```

Comment:
If this is the output of the vision filter, it shouldn't have the camera, just pose/time/visible

Reply:
I'm taking this comment from `RobotState.hpp`, which does include the "which camera" part in the comments:
https://github.com/RoboJackets/robocup-software/blob/fb55e0963d021e8625778f667260addb33dc9031/soccer/WorldState.hpp#L12-L23

Reply:
@kylestach What was the thought behind this?

Reply:
Uh yeah that doesn't sound right. Probably an old comment that didn't get updated, rip.